daemon/src/maker_cfd.rs

```diff
@@ -575,61 +536,23 @@ where
     async fn handle_accept_settlement(&mut self, msg: AcceptSettlement) -> Result<()> {
         let AcceptSettlement { order_id } = msg;
 
-        tracing::debug!(%order_id, "Maker accepts a settlement proposal" );
-
-        let taker_id = self.get_taker_id_of_proposal(&order_id)?;
-
-        match self
-            .takers
-            .send(maker_inc_connections::TakerMessage {
-                taker_id,
-                msg: wire::MakerToTaker::Settlement {
-                    order_id,
-                    msg: wire::maker_to_taker::Settlement::Confirm,
-                },
-            })
-            .await?
-        {
-            Ok(_) => {
-                self.current_agreed_proposals
-                    .insert(order_id, self.get_settlement_proposal(order_id)?);
-                self.remove_pending_proposal(&order_id)
-                    .await
-                    .context("accepted settlement")?;
-            }
-            Err(e) => {
-                tracing::warn!("Failed to notify taker of accepted settlement: {}", e);
-                self.remove_pending_proposal(&order_id)
-                    .await
-                    .context("accepted settlement")?;
-            }
-        }
+        self.settlement_actors
+            .send(&order_id, collab_settlement_maker::Accepted)
+            .await
+            .then(|| ())
```

Comment:
What is this for?

Reply:
`bool -> Option -> Result` so that we can keep the `?`.

Reply:
Should be change the API of `send` then?

Reply:
If you introduce an `ActorName` trait, you could even return a really descriptive error message that actor XYZ is not connected.

Reply:
I tried your suggestions in https://github.com/itchysats/itchysats/pull/775/commits/09ebb963db46cd621e83f78bca32ac08f7543e5a and https://github.com/itchysats/itchysats/pull/775/commits/d2dbb44e7abd4926eac4c0ba0bc7a51cce6b5439.